packages/kit/src/views/Staking/components/UniversalWithdraw/index.tsx

```
@@ -615,7 +632,9 @@ export function UniversalWithdraw({
                     <YStack
                       animation="quick"
                       rotate={
-                        open && !isAccordionTriggerDisabled ? '180deg' : '0deg'
+                        open && !isAccordionTriggerDisabled
+                          ? '180deg'
+                          : '0deg'
```

ESLint

[error] Replace `⏎··························?·'180deg'⏎·························` with `·?·'180deg'`

(prettier/prettier)